README.md

```diff
@@ -59,13 +59,28 @@ dblessConfig:
       - name: ddtrace
         config:
           service_name: kong-ddtrace
-          host: "{vault://env/KONG_DATADOG_AGENT_HOST}"
-          port: "8126"
-          version: "v0.4"
+          agent_host: "{vault://env/KONG_DATADOG_AGENT_HOST}"
           environment: 'dev'
 ...
 ```
 
+### Agent URL
+
+The URL that will be used to connect to the agent. The value should not include the trailing `/` character.
+
+--data 'config.trace_agent_url=http://localhost:8126'
+
+This value can use vault references. By default, the value of the environment variale `DD_TRACE_AGENT_URL` is used by resolving `vault://env/dd-trace-agent-url`.
+When this varuable is not set, the default is `http://localhost:8126`.
+
+### Agent Endpoint (deprecated)
+
+The full URL for submitting traces to the agent. It is preferred to use `agent_host` or `trace_agent_url` options instead.
+This option will be deprecated in future releases.
+
+--data config.agent_endpoint=http://localhost:8126/v0.4/traces'.
```

Comment:
This `--data ...` snippet, and the one above, probably ought to be in backticks like those below.